_includes/gh-buttons.html

```diff
@@ -1,11 +1,11 @@
 <div class="gh-btns">
   <!-- Follow -->
-  <iframe src="http://ghbtns.com/github-btn.html?user=hilios&type=follow"
+  <iframe src="https://ghbtns.com/github-btn.html?user=hilios&type=follow"
```

Comment:
Maybe it's better to use `//` because matches the same protocol the user have use it to access the page, instead of rely on the premise that the user WILL use a secure connection.